### PR TITLE
Add support of new widget on Host Register page for adding multiple repositories

### PR DESCRIPTION
**Pull request by shweta83**
Add support of new widget on Host Register page for adding multiple repositories

### CODE REVIEW

File: airgun/views/host.py [hunk 1]
```diff
@@ -774,7 +786,6 @@ class HostsTaxonomyMismatchRadioGroup(GenericLocatorWidget):
         </form>
     """
 
-    taxonomy = None
```
Comment (shweta83) [inline airgun/views/host.py:777]: The CI checks was failing because of this. I don't know the reason but CI checks started passing after I removed this.

Comment (sambible) [reply airgun/views/host.py:777]: Not familar with this, do you remember the CI check that was failing here? This is only present exactly in this widget, no other place in Airgun.

**Review by sambible — APPROVED** [review]
Code looks good, would like to see it exercised in Robottelo before merge.

File: airgun/views/host.py (hunk 1 above)
Comment (shweta83) [reply airgun/views/host.py:777]: It is failing in make docs with error 
```
Warning, treated as error:
airgun/docs/autoapi/airgun/views/host/index.rst:1845:duplicate object description of airgun.views.host.HostsTaxonomyMismatchRadioGroup.taxonomy, other instance in autoapi/airgun/views/host/index, use :no-index: for one of them

```

**Comment by shweta83** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_registration.py -k test_subscription_manager_install_from_repository

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 326
Build Status: SUCCESS
PRT Comment: pytest tests/foreman/ui/test_registration.py -k test_subscription_manager_install_from_repository --external-logging
Test Result : ========= 6 passed, 13 deselected, 352 warnings in 2280.07s (0:38:00) ==========
```

**Review by omkarkhatavkar — APPROVED** [review]

**Pull request merged**